### ISSUE TITLE
Substantial print pollution when optimising for hyperpameters

### ISSUE THREAD

**Issue by LemurPwned** [sergioburdisso/pyss3#1]
Hello,
while I was trying to optimise for hyperparameters I have found that there is little I could do to avoid having a large printout to stdout.

I tried to mute this by doing the following:
```python
import pyss3
pyss3.Print.set_quiet(True)
```
but besides, it is also `tqdm` that produces progress bars, hence I propose propagating 
`Print.__quiet__ `to `tqdm's` `disable` parameter, for example:
```python
tqdm(..., disable=Print.__quiet__)
```

Thanks :) 
Keep up the good work!

**Comment by sergioburdisso** [comment]
Hi @LemurPwned!
Thanks for this valuable feedback, taking into account your suggestions I've completely redesigned the way in which PySS3 handles verbosity levels. Below I'll paste the message of the commit that solved this issue:

---
In response to Issue #1, the way in which PySS3 handled the level of
verbosity was redesigned in order to give the user greater control
through a clearer interface.
Now PySS3 supports the following 3 levels of verbosity:

- ``0`` (quiet): do not output any message (only error messages)
- ``1`` (normal): default behavior, display only warning messages and
                  progress bars [*]
- ``2`` (verbose): display also the informative non-essential messages

Besides, now the following built-in constants can also be used to refer
to these 3 values: ``VERBOSITY.QUIET``, ``VERBOSITY.NORMAL``, and
``VERBOSITY.VERBOSE``, respectively.
In addition, a ``set_verbosity`` function has been added to the main
module, for the user to change the verbosity level.
For example, let's suppose we want to set the verbosity level to
'quiet', then we could use:

```python
import pyss3
...
pyss3.set_verbosity(0)
...
```
or, equivalently:

```python
import pyss3
from pyss3 import VERBOSITY
...
pyss3.set_verbosity(VERBOSITY.QUIET)
...
```

To carry out this update, the ``Print`` class was partially redesigned,
and therefore, as it can be seen, the source code of all the modules
had to be modified as well (to be compatible with the new ``Print``
design).

[*] Note that, unlike before, PySS3 will now, by default, only show
progress bars (plus warning or error messages).

 ---

I think that this new interface is much clearer and the user can use it without "being aware" of what's "under the hood" (that is, the "Print" class), which I think is great.

Anyway, I was working on a new version release (0.4.0) when I read your issue, so I decided to pospone its release and add this modification to the new version.  This new release will improve and incorporate new features, such as an explicit support for multi-label classification and a method for the user to automatically extract "useful insights" (relevant text fragments) from documents, I'll release this new version today, so you'll be able to update your code after updating the package (I'll notify you as soon as it is released).

:D

Thank you very much for your useful ideas and feedback, buddy :+1:
